### DISCUSSION TITLE
TSL2591 and MLX90614  tested and working!

### DISCUSSION THREAD

**Discussion by Vertikar** [aaronwmorris/indi-allsky#1332] [General]
Hey @aaronwmorris,

Couldn't resist grabbing two more sensors once I'd seen you'd added them and just set them up tonight.

Can see in the logs they're working fine.

Jun 18 23:50:01 allsky [WARNING] MainProcess-1407/Sensor-9 lightSensorTsl2591.__init__() [59]: Initializing TSL2591 I2C light sensor device @ 0x29
Jun 18 23:50:02 allsky [WARNING] MainProcess-1407/Sensor-9 tempSensorMlx90614.__init__() [57]: Initializing MLX90614 I2C temperature device @ 0x5a
Jun 18 23:50:02 allsky [INFO] MainProcess-1407/Sensor-9 lightSensorTsl2591.update() [24]: TSL2591 - lux: 2.0759, visible: 131086, ir: 2, full: 131088
Jun 18 23:50:02 allsky [INFO] MainProcess-1407/Sensor-9 tempSensorMlx90614.update() [21]: MLX90614 - ambient temp: 19.3c, object temp: 20.1c
Jun 18 23:50:05 allsky [WARNING] MainProcess-1407/Sensor-9 sensor.night_day_change() [155]: Day/Night change
Jun 18 23:50:05 allsky [INFO] MainProcess-1407/Sensor-9 lightSensorTsl2591.update() [24]: TSL2591 - lux: 2.0759, visible: 131086, ir: 2, full: 131088
Jun 18 23:50:05 allsky [INFO] MainProcess-1407/Sensor-9 tempSensorMlx90614.update() [21]: MLX90614 - ambient temp: 19.2c, object temp: 20.1c
Jun 18 23:50:05 allsky [INFO] MainProcess-1407/Sensor-9 sensor.saferun() [147]: Sensor SQM: 11.79054

**Comment by aaronwmorris** [comment]
That is fantastic news!  I will update the wiki.
